projects/grafana/openshift-cluster-monitoring.json

```diff
@@ -0,0 +1,5138 @@
+{
```

Comment:
Why is this file different from what you have in https://github.com/openshift/origin/pull/17114/?

Reply:
just few updates.

I need to update that PR as well 10x